*** ./smali/android/preference/PreferenceFrameLayout$LayoutParams.smali	2013-04-18 19:45:12.000000000 +0000
--- /mnt/micode/shooter/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceFrameLayout$LayoutParams.smali	2013-04-18 19:48:39.000000000 +0000
***************
*** 59,64 ****
--- 59,72 ----
  
      iput-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
  
+     iget-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
+ 
+     invoke-static {p1, v1}, Landroid/preference/PreferenceFrameLayout$LayoutParams$Injector;->isRemoveBorders(Landroid/content/Context;Z)Z
+ 
+     move-result v1
+ 
+     iput-boolean v1, p0, Landroid/preference/PreferenceFrameLayout$LayoutParams;->removeBorders:Z
+ 
      invoke-virtual {v0}, Landroid/content/res/TypedArray;->recycle()V
  
      return-void
